src/elements/input.ts

```diff
@@ -4,12 +4,11 @@ import { customElement, property } from 'lit/decorators.js'
 @customElement('hangman-input')
 export class HangmanInput extends LitElement {
   @property()
```

Comment:
i think you forgot to remove the `@property()` could you delete it?

Reply:
true